lib/engine/game/g_1848/round/operating.rb

```diff
@@ -16,6 +16,14 @@ def after_process(action)
 
             super
           end
+
+          def skip_entity?(entity)
+            return super if entity.name != :COM
+
+            com_operates = @game.sydney_adelaide_connected || @game.com_can_operate
+            @log << 'COM does not operate, Sydney and Addelaide are not connected or phase 6 has not started' unless com_operates
```

Comment:
Lots of negative statements in the log message. Would it read better as COM does not operate until either Sydney to Addalaide is connected or phase 6 has started?